### ISSUE TITLE
Not everything should be a property in TextAnnotation

### ISSUE THREAD

**Issue by shyamupa** [CogComp/cogcomp-nlpy#112]
`ta.get_pos` and `ta.get_ner` should be vanilla methods instead of being properties. 

For properties, I suggest have `ta.pos` and `ta.ner` if we want that functionality.

**Comment by nitishgupta** [comment]
I also agree with Shyam. I don't see a point of having ta.pos style properties. Just functions should suffice. But, if we do so now, a lot of old code will break. What should we do. I don't know of code apart from Lorelei and My-EL that uses cogcomp-nlpy. So this might be a good time before more people start to use this.
